README.md

```diff
@@ -83,7 +83,7 @@ See [variables.tf] and [examples/] for details and use-cases.
 
   A unique ID for this dataset, without the project name.
 
-- **`role`**: **_(Required `string`)_**
+- **`role`**: _(Optional `string`)_
 
```

Comment:
role is not optional in code?

Reply:
@mariux  it is, i can pass only policy_bindings